R/query-string.R

```diff
@@ -71,10 +71,11 @@ createPathRegex <- function(pathDef){
 
 typeToRegex <- function(type){
   re <- rep("[^/]+", length(type))
```

Comment:
It would help to have a comment explaining which cases get this default value (like `NA` and `"chr"`?)